webdriver/tests/classic/perform_actions/wheel.py

```diff
@@ -55,6 +55,12 @@ def test_scroll_iframe(session, test_actions_scroll_page, wheel_chain):
 
     wheel_chain.scroll(0, 0, 5, 10, origin=target).perform()
 
+    # Chrome requires some time to process the event from the iframe, so we wait for it.
+    while True:
+        events = get_events(session)
```

Comment:
would it spin too frequently?

Reply:
I don't think so. It makes a webdriver classic request.

Reply:
In my case it make 6 requests until the events are in place

Reply:
and the event is there in 10 ms

Reply:
well, we would need some sort of timeout here too. Can we use asyncpoll here as well?

Reply:
it's sync, so we can add `time.sleep(0.1)`

Reply:
> well, we would need some sort of timeout here too. Can we use asyncpoll here as well?

did it with time. Looks odd as well )